moztelemetry/zeppelin.py

```diff
@@ -0,0 +1,25 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+
+from StringIO import StringIO
+
+
+def show(fig):
+    """ Renders a Matplotlib figure in Zeppelin.
+
+    :param fig: a Matplotlib figure
+
+    Usage example::
+
+        import matplotlib.pyplot as plt
+        from moztelemetry.zeppelin import show
+
+        fig = plt.figure()
+        plt.plot([1, 2, 3])
+        show(fig)
+    """
+    img = StringIO()
+    fig.savefig(img, format='svg')
+    img.seek(0)
+    print("%html <div style='width:600px'>" + img.buf + "</div>")
```

Comment:
It may be good to add a `width` parameter to let the user customize the image size (default to 600px sounds good to me).